source/reference/verification.txt

```diff
@@ -0,0 +1,211 @@
+   work for standalone servers and replica sets that use MongoDB 4.4 or earlier
+   releases, since the field order can vary.
+
+To verify that the source and destination cluster have the same hashes, execute
+the :dbcommand:`dbHash` command for the synced collections on each cluster and
+compare the hash value in the ``md5`` field of the return documents.
+
+.. io-code-block::
+
+   .. input::
+      :language: javascript
+
+      db.runCommand({
+         dbhash: 1,
+         collections: [
+            "accounts.us_accounts",
+            "accounts.eu_accounts",
+            ...
+         ]}).md5
+
+   .. output::
+      :language: text
+
+      d41d8cd98f00b204e9800998ecf8427e
+
+.. _c2c-verify-doc-comp:
+
+Document Comparison
+-------------------
+
+You can verify sync by comparing documents on the source and destination
+clusters. Write a script that queries collections on the source cluster and
+then checks that the document exists with the same values on the destination
+cluster.
+
+.. _c2c-verify-migration-verifier:
+
+Migration Verifier
+------------------
+
+Migration Verifier connects to the source and destination clusters and performs
+a series of verification checks, comparing documents, views, and indexes to
+confirm the sync was successful.
+
+.. important::
+
+   Migration Verifier is an experimental and unsupported tool.
+
+To use the Migration Verifier,
+
+#. Start Migration Verifier:
+
+   .. code-block:: bash
+
+      migration-verifier --verifyAll \
+         --srcURI example.net:27020 \
+         --destURI example.net:27021 \
+         --metaURI example.net:27017
+
+
+#. To start the verification checks, use cURL to send the ``check`` command to
+   Migration Verifier:
+
+   .. code-block:: bash
+
+      curl -H "Content-Type: application/json" \
+          -X POST -d '{}' http://127.0.0.1:27020/api/v1/check
+
+#. When ``mongosync`` is complete, use cURL to send the ``writeOff`` command to
+   Migration Verifier:
+
+   .. code-block:: bash
+
+      curl -H "Content-Type: application/json" \
+          -X POST -d '{}' http://127.0.0.1:27020/api/v1/writesOff
+
+
+#. To view the results, use cURL to send the ``progress`` command to Migration
+   Verifier:
+
+   .. io-code-block::
+
+      .. input::
+         :language: bash
+
+         curl -H "Content-Type: application/json" \
+            -X GET http://127.0.0.1:27020/api/v1/progress
+
+      .. output::
+         :language: json
+
+         {
+            "progress": {
+               "phase": "idle",
+               "error": null,
+               "verificationStatus": {
+                  "totalTasks": 1,
+                  "addedTasks": 0,
+                  "processingTasks": 0,
+                  "failedTasks": 1,
+                  "completedTasks": 0,
+                  "metadataMismatchTasks": 0,
+                  "recheckTasks": 0
+               }
+            }
+         }
+
+
+When the ``phase`` field returns ``idle``, the verification checks are
+complete.
```

Comment:
Can we be specific about what the user should be checking in the output to confirm whether or not the sync was successful?

Reply:
Fixed